### PR TITLE
Feat/prove with pdf + pwd

### PR DESCRIPTION
**Pull request by Meyanis95**
## Motivation

Creating the V2 Login flow, with the corresponding components and methods

## Change Summary

- Created new components under the `components/V2` folder + exporting `LogInWithAnonAadhaarV2` comp.
- Implemented the extractWitness util, to handle pdf treatment and witness generation

## Merge Checklist

_Choose all relevant options below by adding an `x` now or at any time before submitting for review_

- [X] PR title adheres to the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) standard
- [X] PR has a [change set](https://github.com/privacy-scaling-explorations/anon-aadhaar/CHANGELOG.md)
- [X] PR has been tagged with a change label(s) (i.e. documentation, feature, bug fix, or chore)
- [X] PR includes documentation if necessary.

## Additional Context

If this is a relatively large or complex change, provide more details here that will help reviewers

### CODE REVIEW

File: packages/anon-aadhaar-react/src/extractWitness.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
+import { extractCert, extractSignature } from 'anon-aadhaar-pcd'
```
Comment (vuvoth) [inline packages/anon-aadhaar-react/src/extractWitness.ts:1]: I think this function should move to pcd package.

**Review by vuvoth — COMMENTED** [review]
I leaved a few comments.

File: packages/anon-aadhaar-react/src/extractWitness.ts (hunk 1 above)
Comment (Meyanis95) [reply packages/anon-aadhaar-react/src/extractWitness.ts]: Done!

File: packages/anon-aadhaar-react/src/interface.ts [hunk 2]
```diff
@@ -16,6 +16,12 @@ export enum AadhaarSignatureValidition {
   ERROR_VERIFYING_SIGNATURE = 'An error occurred when verifying signature ❌',
 }
 
+export type Witness = {
```
[the remark this answers is not in the document]
Comment (Meyanis95) [reply packages/anon-aadhaar-react/src/interface.ts]: Done!

**Review by vuvoth — APPROVED** [review]
LGTM!

**Pull request merged**